src/node.cpp

```diff
@@ -477,9 +489,11 @@ bool Node::serialize(string* d)
         d->append(fileattrstring.c_str(), ll);
     }
 
+    bool hasLinkCreationTs = true;
     char isExported = plink ? 1 : 0;
     d->append((char*)&isExported, 1);
-    d->append("\0\0\0\0\0\0", 7);
+    d->append((char*)&hasLinkCreationTs, 1);
```

Comment:
I think it would be better to declare `hasLinkCreationTs` as a `char`, rather than a `boolean`. Later you will read it as char, so...
Also, please, declare variables next to their usage.
I'll fix it